azure-pipelines-integration.yml

```diff
@@ -18,22 +18,23 @@ jobs:
 - job: VS_Integration
   pool:
     name: NetCorePublic-Pool
-    queue: buildpool.windows.10.amd64.vs2019.pre.open
+    queue: buildpool.windows.vs2019.pre.scouting.open
```

Comment:
Do we still need this or is the Preview 2 queue now the main queue? We don't want to be permanently on the scouting queue or else we'd be get broken the moment it's updated.

Reply:
(fine to merge in this state if it's necessary but please fix ASAP as otherwise this is setting a trap to break things)

Reply:
Preview2 is the main queue now, which is coincidentally why the integration tests aren't quite working
https://github.com/dotnet/core-eng/wiki/VS2019-Upgrade-Schedule

Reply:
Moved back to the main pool.